lib/SPIRV/SPIRVWriter.cpp

```diff
@@ -1236,6 +1237,18 @@ void addIntelFPGADecorations(
       E->addDecorate(
           new SPIRVDecorateMergeINTELAttr(E, Name.str(), Direction.str()));
     } break;
+    case DecorationBankBitsINTEL: {
+      SmallVector<StringRef, 1> A;
+      StringRef(I.second).split(A, ',');
+      std::vector<SPIRVWord> Bits(A.size());
+      SPIRVWord Result;
+      for (size_t J = 0; J < A.size(); ++J) {
+        Result = 0;
+        A[J].getAsInteger(10, Result);
```

Comment:
Will it work?
```suggestion
        A[J].getAsInteger(10, Bits[J]);
```

Reply:
Yes, it will work if we don't use range-based loop